### PR TITLE
Kippo fixes

### PR DESCRIPTION
**Pull request by jt6211**

### CODE REVIEW

**Comment by Epicism** [comment]
Thank you. For existing Kippo boxes do we just re-deploy the script? Or is there an upgrade path?

**Comment by jt6211** [comment]
@Epicism this work:

On your MHN web app, copy the contents of the new kippo deploy script (https://raw.githubusercontent.com/threatstream/mhn/master/scripts/deploy_kippo.sh) into the kippo deploy script on your MHN web app and then click "Update".

on your kippo box:
```
sudo supervisorctl stop kippo
sudo rm -rf /opt/kippo
sudo userdel kippo
sudo rm /etc/supervisor/conf.d/kippo*
sudo supervisorctl update
```

Now, on yor kippo box, paste the deploy command from your MHN deploy page for the updated kippo script.

**Comment by Epicism** [comment]
Awesome, thank you.

File: scripts/deploy_kippo.sh [hunk 1]
```diff
@@ -71,31 +91,40 @@ secret = $HPF_SECRET
...
-    echo "Adding iptables port forwarding rule...\n"
-    iptables -F -t nat
-    iptables -t nat -A PREROUTING -i eth0 -p tcp -m tcp --dport 22 -j DNAT --to-destination 127.0.0.1:64222
-fi
+
+# Fix permissions for kippo
+chown -R kippo:users /opt/kippo
+touch /etc/authbind/byport/22
+chown kippo /etc/authbind/byport/22
+chmod 777 /etc/authbind/byport/22
```
Comment (radekk) [inline scripts/deploy_kippo.sh]: On Ubuntu 14.04 ``chmod 777`` didn't worked for me, ``chmod 750`` was fine.

File: scripts/deploy_kippo.sh [hunk 2]
```diff
@@ -109,7 +138,7 @@ user=kippo
 stopasgroup=true
 EOF
 else
-cat > /etc/supervisor/conf.d/kippo.conf <<EOF
+    cat > /etc/supervisor/conf.d/kippo.conf <<EOF
 [program:kippo]
 command=/opt/kippo/start.sh
```
Comment (radekk) [inline scripts/deploy_kippo.sh]: I need to change it to: ``command=authbind --deep /opt/kippo/start.sh``

Comment (radekk) [inline scripts/deploy_kippo.sh]: I see that it was added to ``start.sh``, it should work fine. Thx.

**Pull request merged**